.github/workflows/verifyTemplates.yml

```diff
@@ -0,0 +1,24 @@
+name: Validate ImageStream templates
+
+on:
+  push:
+    branches: [ "release", "OPENJDK-2048-imagestream-scripts" ]
+  pull_request:
+    branches: [ "release", "OPENJDK-2048-imagestream-scripts" ]
```

Comment:
`OPENJDK-2048-imagestream-scripts` branch mention shouldn't be part of the change? Verification could be done on a derivative branch, right?

Reply:
Once merged we definitely don't need mention of this branch. And we might not now, either. I wasn't sure (still not) whether `on.pull_request.branches` refers to the source (`OPENJDK-2048-imagestream-scripts`) or target branch (`release`). If the former, we need this pre-merge to run the job in the PR itself. But if I look at the docs more closely (I spent a lot of time with them last week for some recent `jlink-dev` changes) I can clarify that.

Reply:
I've reviewed the docs, and the branch specified here is the target branch, so `release` is valid and `OPENJDK-2048-imagestream-scripts` superfluous. I've updated the PR accordingly.